FFMpegCore/FFMPEG/FFMpeg.cs

```diff
@@ -436,14 +436,21 @@ public VideoInfo ReplaceAudio(VideoInfo source, FileInfo audio, FileInfo output,
 
         public VideoInfo Convert(ArgumentContainer arguments)
         {
-            var args = ArgumentBuilder.BuildArguments(arguments);
             var output = ((OutputArgument)arguments[typeof(OutputArgument)]).GetAsFileInfo();
+            var sources = ((InputArgument)arguments[typeof(InputArgument)]).GetAsVideoInfo();
+
+            // Sum duration of all sources
+            _totalTime = TimeSpan.Zero;
+            foreach (var source in sources)
+                _totalTime += source.Duration;
 
             if (!RunProcess(arguments, output))
             {
                 throw new FFMpegException(FFMpegExceptionType.Operation, "Could not replace the video audio.");
             }
 
+            _totalTime = TimeSpan.MinValue;
+
```

Comment:
Why is `_totalTime` reset before return?

Reply:
Actually i was unsure as well and copied it from the other convert method.
Maybe just remove both?

Reply:
No, just keep it. I just asked to hear if there was a reason for it being there :)